README.md

```diff
@@ -1,6 +1,25 @@
+![LOGO](https://user-images.githubusercontent.com/62508697/139729943-d46fc1ee-9344-4563-9d7d-2369f38fce54.jpeg)
+
+
+
 # Falcon
 
 Manage and control workflow app
+# Description
+
+Our product is design to help you control your team work better and easier.
+The product allows you as a manager to create new tasks and assigns it to your team, and for your workers, to get 
```

Comment:
The word "workers" is not in common use these days, I think. Even "employees" is not the best choice. What do you think about "team members"?